src/main/java/com/minecolonies/coremod/entity/citizen/citizenhandlers/CitizenSleepHandler.java

```diff
@@ -172,7 +172,7 @@ private void notifyCitizenHandlersOfWakeUp()
     private void spawnCitizenFromBed()
     {
         final BlockPos spawn;
-        final BlockState bedState = citizen.level.getBlockState(getBedLocation());
+        final BlockState bedState = getBedLocation().equals(BlockPos.ZERO) ? null : citizen.level.getBlockState(getBedLocation());
         if (!getBedLocation().equals(BlockPos.ZERO) && bedState.is(BlockTags.BEDS))
```

Comment:
can just make this a null check then

Reply:
technically ye, but easier to read like this